docs/developer-docs/latest/guides/registering-a-field-in-admin.md

```diff
@@ -381,89 +246,156 @@ const Editor = ({ onChange, name, value }) => {
 
-::: tab yarn
+        newValue = `${newValue}${imgTag}`
+      }
 
-```
-yarn build
-```
+      // Handle videos and other type of files by adding some code
+    });
 
-:::
+    onChange({ target: { name, value: newValue } });
+    handleToggleMediaLib();
+  };
+  
+  return (
+    <>
+      <Stack size={1}>
+        <Box>
+          <Typography variant="pi" fontWeight="bold">
+            {formatMessage(intlLabel)}
+          </Typography>
+          {required && 
+            <Typography variant="pi" fontWeight="bold" textColor="danger600">*</Typography>
+          }
+        </Box>
+        <Button startIcon={<Landscape />} variant='secondary' fullWidth onClick={handleToggleMediaLib}>Media library</Button>
+        <Editor 
+          disabled={disabled} 
+          name={name} 
+          onChange={onChange} 
+          value={value} 
+        />
+        {error && 
+          <Typography variant="pi" textColor="danger600">
+            {formatMessage({ id: error, defaultMessage: error })}
+          </Typography>
+        }
+        {description && 
+          <Typography variant="pi">
+            {formatMessage(description)}
+          </Typography>
+        }
+      </Stack>
+      <MediaLib 
+        isOpen={mediaLibVisible} 
+        onChange={handleChangeAssets}
+        onToggle={handleToggleMediaLib} 
+      />
+    </>
+  );
+};
 
-::: tab npm
+Wysiwyg.defaultProps = {
+  description: '',
+  disabled: false,
+  error: undefined,
+  intlLabel: '',
+  required: false,
+  value: '',
+};
 
-```
-npm run build
+Wysiwyg.propTypes = {
+  description: PropTypes.shape({
+    id: PropTypes.string,
+    defaultMessage: PropTypes.string,
+  }),
+  disabled: PropTypes.bool, 
+  error: PropTypes.string, 
+  intlLabel: PropTypes.shape({
+    id: PropTypes.string,
+    defaultMessage: PropTypes.string,
+  }),
+  name: PropTypes.string.isRequired,
+  onChange: PropTypes.func.isRequired,
+  required: PropTypes.bool,
+  value: PropTypes.string, 
+};
+
+export default Wysiwyg;
 ```
 
 :::
 
-::: tab strapi
+## Register the field
 
-```
-strapi build
-```
+The last step is to register the field `wysiwyg` with the WYSIWYG component with `addFields()`.
 
-:::
+```js
+// path: wysiwyg/admin/src/index.js
 
-::::
+import pluginPkg from "../../package.json";
+import Wysiwyg from "./components/Wysiwyg";
+import pluginId from "./pluginId";
 
-::: tip
-If the plugin still doesn't show up, you should probably empty the `.cache` folder too.
-:::
+const name = pluginPkg.strapi.name;
+
+export default {
+  register(app) {
+    app.addFields({ type: 'wysiwyg', Component: Wysiwyg });
+
+    app.registerPlugin({
+      id: pluginId,
+      isReady: true,
+      name,
+    });
+  },
+  bootstrap() {},
+};
+```
 
 And VOILA, if you create a new `collectionType` or a `singleType` with a `richtext` field you will see the implementation of [CKEditor](https://ckeditor.com/ckeditor-5/) instead of the default WYSIWYG.
```

Comment:
```suggestion
And _voilà_, if you create a new `collectionType` or a `singleType` with a `richtext` field you will see the implementation of [CKEditor](https://ckeditor.com/ckeditor-5/) instead of the default WYSIWYG.
```

It's OK to be a bit more friendly here, but we should avoid using caps :-) 
I've added italics since it's a French word in an English sentence.